common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -292,10 +263,18 @@ $link-color: rgb(26, 161, 222) !default;
 
 nav.sequence-bottom {
   position: relative;
-  width: 79px;
-  height: 1px;
+  height: 45px;
   margin: lh(2) auto;
-  text-align: center;
+  display: flex;
+  justify-content: center;
```

Comment:
We support only IE11: http://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/front_matter/browsers.html